src/components/NewsletterSignup/NewsletterSignup.test.tsx

```diff
@@ -0,0 +1,332 @@
+import { render, screen } from "@testing-library/react";
+//import userEvent from "@testing-library/user-event";
+import { axe } from "jest-axe";
+import * as React from "react";
+import renderer from "react-test-renderer";
+import NewsletterSignup from "./NewsletterSignup";
+import { sectionDataMap } from "../../helpers/types";
+
+// If you want to see what's happening, insert below render()
+// screen.debug();
+
+describe("NewsletterSignup Accessibility", () => {
+  const onSubmit = jest.fn(),
+    onChange = jest.fn(),
```

Comment:
Nitpick: our convention is to declare every variable on a new line with a new `const`.

Reply:
Changed.